fastchat/model/apply_delta.py

```diff
@@ -2,33 +2,69 @@
 Apply the delta weights on top of a base model.
 
 Usage:
-python3 -m fastchat.model.apply_delta --base ~/model_weights/llama-13b --target ~/model_weights/vicuna-13b --delta lmsys/vicuna-13b-delta-v1.1
+python3 -m fastchat.model.apply_delta --base ~/model_weights/llama-13b --target ~/model_weights/vicuna-13b --delta lmsys/vicuna-13b-delta
 """
 import argparse
 
 import torch
 from tqdm import tqdm
 from transformers import AutoTokenizer, AutoModelForCausalLM
-
+from transformers import LlamaTokenizer, AutoConfig
+import gc
+import shutil
 
 def apply_delta(base_model_path, target_model_path, delta_path):
-    print(f"Loading the base model from {base_model_path}")
-    base = AutoModelForCausalLM.from_pretrained(
-        base_model_path, torch_dtype=torch.float16, low_cpu_mem_usage=True)
-    base_tokenizer = AutoTokenizer.from_pretrained(
-        base_model_path, use_fast=False)
 
     print(f"Loading the delta from {delta_path}")
-    delta = AutoModelForCausalLM.from_pretrained(delta_path, torch_dtype=torch.float16, low_cpu_mem_usage=True)
+    delta_tokenizer = AutoTokenizer.from_pretrained(delta_path, use_fast=False)
+
+
+    DEFAULT_PAD_TOKEN = "[PAD]"
+    base_tokenizer = AutoTokenizer.from_pretrained(base_model_path, use_fast=False)
+    num_new_tokens = base_tokenizer.add_special_tokens(dict(pad_token=DEFAULT_PAD_TOKEN))
+    print("num_new_tokens: ",num_new_tokens)
```

Comment:
These are not required, please update according to https://github.com/lm-sys/FastChat/blob/abf7ca40e1f84e00436ec1daa42b57a2a378327f/fastchat/model/apply_delta.py#L14-L31.